testing/other/CMakeLists.txt

```diff
@@ -13,8 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #=============================================================================
-add_cmake_config_test( rapids_cmake-multiple-cpm )
-add_cmake_config_test( rapids_cmake-multiple-simple )
+add_cmake_config_test( rapids_cmake-multiple-cpm NO_RAPIDS_CMAKE_HOOKS)
+add_cmake_config_test( rapids_cmake-multiple-simple NO_RAPIDS_CMAKE_HOOKS)
 
-add_cmake_config_test( FetchContent-legacy )
-add_cmake_config_test( FetchContent-hostile-legacy )
+add_cmake_config_test( FetchContent-legacy NO_RAPIDS_CMAKE_HOOKS)
+add_cmake_config_test( FetchContent-hostile-legacy NO_RAPIDS_CMAKE_HOOKS)
```

Comment:
Why are we now disabling these hooks? We used to do the fetch emulation for every test IIUC, why remove it for these now?

Reply:
The hooks this is disabling is the emulation that rapids-cmake has already been fetched ( added by testing/utils/emulate_fetching_rapids_cmake.cmake ).

Previous to this PR, directories would not have any rapids-cmake hook emulation automatically added, while simple `.cmake` files would.  

In fixing this oversight  needed to disable this hook emulation for these tests as they can verify that can actually fetch rapids-cmake in the legacy manner.